language/ko.json

```diff
@@ -0,0 +1,97 @@
+{
+  "semantics": [
+    {
+      "label": "안내",
+      "description": "학생들이 이 안내사항을 노트 위에 볼 것입니다."
+    },
+    {
+      "label": "연습 내용"
+    },
+    {
+      "label": "노트 영역",
+      "description": "노트 영역 설정",
+      "fields": [
+        {
+          "label": "영역 제목 불러오기",
+          "default": "불러오기",
+          "description": "불러오기 영역 제목"
+        },
+        {
+          "label": "불러오기 영역 자리",
+          "default": "키워드, 질문, 요지 등을 입력하세요.",
+          "description": "학생들에게 상기시키기 위한 영역 자리"
+        },
+        {
+          "label": "노트 영역 제목",
+          "default": "노트",
+          "description": "노트 영역에 대한 제목"
+        },
+        {
+          "label": "노트 영역 자리",
+          "default": "일자, 세부사항, 정의, 공식, 예시 등 입력",
+          "description": "학생들에게 상기시키기 위한 영역 자리"
+        },
+        {
+          "label": "요약 영역 제목",
+          "default": "요약",
+          "description": "요약 영역 제목"
+        },
+        {
+          "label": "요약 영역 자리",
+          "default": "요약을 입력하세요",
+          "description": "학생들에게 상기시키기 위한 영역 자리"
+        }
+      ]
+    },
+    {
+      "label": "행동 설정",
+      "fields": [
+        {
+          "label": "시작 시 노트 표시",
+          "description": "활성화되면 시작 시 노트가 표시됩니다."
+        }
+      ]
+    },
+    {
+      "label": "읽어주기 기능 텍스트",
+      "fields": [
+        {
+          "label": "전체 화면 버튼 제목",
+          "default": "전체 화면 모드로 전환"
+        },
+        {
+          "label": "전체 화면 버튼 제목(종료)",
+          "default": "전체 화면 모드 종료"
+        },
+        {
+          "label": "토글(Toggle) 버튼 제목(노트가 닫혀 있을 때)",
+          "default": "노트 열기"
+        },
+        {
+          "label": "토글(Toggle) 버튼 제목(노트가 열려 있을 때)",
+          "default": "노트 닫기"
+        },
+        {
+          "label": "노트가 열렸을 때",
+          "default": "보기가 노트로 전환되었습니다."
+        },
+        {
+          "label": "노트가 닫혔을 때",
+          "default": "보기가 연습 문제로 전환되었습니다."
+        }
+      ]
+    }
+  ]
+}
+© 2021 GitHub, Inc.
```

Comment:
Please remove those lines. Feel free to use https://translate-h5p.tk/ to translate H5P content types. It's not prone to those errors and offers other benefits as well.

Reply:
Hi, @otacke. I find this translation site, https://translate-h5p.tk/, is inconvenient for me. I will make sure that I submit immaculate code. Thank you. I updated the code above.

Reply:
Whatever ia better for you. Have checked your latest changes with a JSON checker?

Reply:
Hi, @otacke, I updated the code and checked the updated version with a JSON checker. I hope this version is all right. Thank you.